dbuf-gen/tests/samples/sample_2.rs

```diff
@@ -0,0 +1,200 @@
+// this is rough sketch of some file that should be generated by codegen
+
+// https://github.com/SphericalPotatoInVacuum/DependoBuf/blob/main/test/cpp_test_samples/dbuf_files/rt_dependent_messages.dbuf
+
+// message Sum (a Int) {}
+
+// message Foo (a Int) (b Int) {
+//     sum Sum (-a + b);
+// }
+
+// message Bar (c Int) (s String) {
+//     e Int;
+//     d Int;
+//     f Foo e d;
+//     g Foo c (e + d);
+// }
+
+// message Kek (a Int) (b Int) (f Foo a b) {
+//     bar Bar a "qwe";
+// }
+
+#[derive(Debug)]
+pub enum ConstructorError {
+    MismatchedDependencies,
+}
+
+#[derive(PartialEq, Eq)]
+pub struct Message<Body, Dependencies> {
+    pub body: Body,
+    pub dependencies: Dependencies,
+}
+
+// actual generated part
+
+// prelude
+pub type Box<T> = std::boxed::Box<T>;
```

Comment:
`Box` is already provided by the `dbuf-rust-runtime`, is the usage here intentional?